tiller_python_service/git_actions.py

```diff
@@ -260,6 +260,9 @@ def parse_pull_request_url(pull_request_url):
     else:
         return None, None, None
 
+def totally_cool_change(variable):
```

Comment:
tiller help "totally_cool_change" should be named "real_cool_change" and "variable" should be "change"

Reply:
help tiller "totally_cool_change" should be named "real_cool_change" and "variable" should be "change"

Reply:
help tiller "totally_cool_change" should be named "real_cool_change" and "variable" should be "change"

Reply:
lgtm tiller